elastic/kibana
# [Synthetics] notify when no default connectors are present

## Summary

Closes https://github.com/elastic/kibana/issues/152431

Adds a notice when no default connectors is configured in two places.
1. On the overview page
2. On the monitor edit page (when alerting is enabled for that monitor)

<img width="1408" alt="Screen Shot 2023-04-06 at 3 38 00 PM" src="https://user-images.githubusercontent.com/11356435/231041883-a250ccd2-58d6-4882-a425-4b8c09cc194f.png">
<img width="1270" alt="Screen Shot 2023-04-06 at 3 37 49 PM" src="https://user-images.githubusercontent.com/11356435/231041884-3fa70b3c-8aad-4b94-9700-d0d58885ad08.png">

### Testing

1. Ensure you have no default connectors set
2. Create a monitor
3. Navigate to the overview page. Observe the notice
4. Navigate to the edit monitor page for that montior. Observe the notice.
5. Create a second monitor. This time, disable alerting for that monitor
6. Navigate to the edit page for the second monitor. Ensure that the notice does not appear.
